src/components/diffViewer/DiffViewer.tsx

```diff
@@ -0,0 +1,43 @@
+import {Diff2Html} from 'diff2html';
+import * as React from 'react';
+import * as unidiff from 'unidiff';
+import * as styles from './styles/DiffViewer.scss';
+
+export const OutputFormat = {

```

Comment:
These const should go in the `DiffViewer` class as static variables IMO, because they're only relevant in that context. Also, this will reduce the number of exported variabled.